doc/nrf/gs_assistant.rst

```diff
@@ -36,31 +42,63 @@ The app is installed on your machine, and the :guilabel:`Install` button changes
 
 .. _gs_app_installing-ncs-tcm:
 
-Installing the |NCS|
-====================
+.. rst-class:: numbered-step
+
+Install the |NCS|
+*****************
+
+Complete the following steps to install the |NCS| source code:
 
-Once you have installed the Toolchain Manager, open it in nRF Connect for Desktop.
+1. Open Toolchain Manager in nRF Connect for Desktop.
```

Comment:
Should we update the following image to Toolchain Manager v0.10.3 to match with the second screenshot?